staff/aleix-caminal/cv-styled/assets/css/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  display: grid;
+  grid-template: auto auto/1fr 260px;
+  grid-gap: 1.25rem;
+  padding: 1.25rem;
+  font-family: TT Commons, system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Droid Sans, Helvetica Neue;
+}
+body > header {
+  display: grid;
+  grid-column: 1;
+  grid-row: 1;
+  grid-template: auto/auto 1fr;
+  grid-gap: 2.5rem;
+  height: auto;
+}
+body > main {
+  grid-column: 1;
+  grid-row: 2;
+}
+body > aside {
+  grid-column: 2;
+  grid-row: 1/3;
+}
+
+.text--hl, a, time {
+  color: #2196f3;
+}
+
+a {
+  text-decoration: none;
+  transition: all 0.4s;
+}
+a:hover {
+  color: #0960a5;
+  transition: all 0.2s;
+}
+
+.button {
+  display: inline-block;
+  padding: 0.75rem 1rem;
+  border-radius: 1.5rem;
+  text-decoration: none;
+  font-size: 1.25rem;
+  background-color: #2196f3;
+  color: #fff;
+  transition: all 0.4s;
+}
+.button:hover {
+  color: #fff;
+  background-color: #0960a5;
+  transition: all 0.2s;
+}
+
+.text--mb {
+  margin-bottom: 0.25rem !important;
```

Comment:
Normally !important is used as a last resort. Is there no other way of ensuring this style will be applied?

Reply:
there's really no other way to do it :/